### PR TITLE
fix(macOS): Window sizing

### LINKED ISSUE #5979: [macOS] window is not resizable
## Current behavior

can't resize window by using left bottom and right bottom corners
(it works with top left & right ones)

## Expected behavior

enable resizing 

## How to reproduce it (as minimally and precisely as possible)
1. create a new Uno project
2. execute
## Environment

Nuget Package:

- [X] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia version 3.7.3
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [X] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [ ] Visual Studio 2019 (version: )
- [X] Visual Studio for Mac (version: 8.9.8 build 7)
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

Relevant plugins:

- [ ] Resharper (version: )

### LINKED ISSUE #6330: macOS Main Window cropped at launch and can't resize it with the mouse.
## Current behavior
After building a new macOS application the right side and sometime the bottom of the application is cropped and you can't grab the edge of the application to resize it.  You can Maximize it and then the missing area will appear but Resizing manually is broken now.

## Expected behavior
Show the full application and allow re-sizing to work like any normal application....hehe. ;-)

Show the full application and allow re-sizing to work like any normal application....hehe. ;-)

Nuget Package:

- [ ] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s):
3.9.0 dev 83

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [x] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [ ] Visual Studio 2019 (version: )
- [x] Visual Studio for Mac (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

## Anything else we need to know?

Any new maOS app should also allow Command+Q Command+H and Command+W all to works out the gate. for Quit, Hie and Close.  That is just basic things any new XCode app would create for you also by default for obvious reasons...  ;-)

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): #

fixes #5979, fixes #6330 

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?
Unable to resize Window from the bottom edges

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/Controls/Window.macOS.cs [hunk 1]
```diff
@@ -129,11 +129,9 @@ public override void SendEvent(NSEvent evt)
 					case NSEventType.ScrollWheel:
 						CoreWindowEvents?.RaisePointerWheelChanged(ToPointerArgs(evt, posInWindow));
 						break;
-
-					default:
-						base.SendEvent(evt);
```
Comment (davidjohnoliver) [inline src/Uno.UI/Controls/Window.macOS.cs:134]: Oops!

**Review by davidjohnoliver — APPROVED** [review]

**Review by MartinZikmund — APPROVED** [review]

File: src/Uno.UI/Controls/Window.macOS.cs (hunk 1 above)
Comment (ajpinedam) [reply src/Uno.UI/Controls/Window.macOS.cs:134]: Yeah @davidjohnoliver, we don't want a default handler in the `case` instead we want to call the base all the time.

Comment (jeromelaban) [reply src/Uno.UI/Controls/Window.macOS.cs:134]: There isn't any need to filter the events based on the actual usage in uno ?

Comment (ajpinedam) [reply src/Uno.UI/Controls/Window.macOS.cs:134]: Yesterday when synced with @dr1rrb we agreed to call base for all the different events (without filtering) and check if this brings any performance hit as we could be executing the pointer event twice. 

I ran the sample app with this chance and did not notice any improvement degradation.

@jeromelaban

**Review by dr1rrb — APPROVED** [review]

**Pull request merged**